docs/management/admin/blocklist.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[blocklist]]
+[chapter]
+= Blocklist
+
+coming[8.2.0]
+
+The blocklist allows you to prevent specified applications from running if you're certain they should never run in a host environment, extending the list of processes that {endpoint-sec} considers malicious. This is especially useful to ensure that known malicious processes aren't accidentally executed by an end user. 
+
+[NOTE] 
+=====
+In addition to configuring specific entries on the **Blocklist** page, you must also ensure that the blocklist is enabled on the {endpoint-sec} integration policy in the <<malware-protection, Malware protection settings>>. This setting is enabled by default.
+
+You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
+=====
+
+By default, a blocklist entry is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a blocklist entry to a specific {endpoint-sec} integration policy, which blocks the process only on hosts assigned to that policy.
+
+. Go to **Manage** -> **Blocklist**.
+
+. Click **Add blocklist entry**. The **Add blocklist** flyout is displayed.
+
+. Fill in these fields in the **Details** section:
+.. `Name`: Enter a name to identify the blocklist entry.
+.. `Description`: Enter a description to provide more information on the blocklist entry (optional).
+
+. In the **Conditions** section, enter the following information about the application you want to block:
+.. `Select operating system`: Select the appropriate operating system from the drop-down.
+.. `Field`: Select a field to identify the application being blocked:
+  * `Hash`: The MD5, SHA-1, or SHA-256 hash value of the application's executable.
+  * `Path`: The full file path of the application's executable.
+  * `Signature`: (Windows only) The name of the application's digital signer.
++
+TIP: To find the signer's name for an application, go to *Kibana* -> *Discover* and query the process name of the application's executable (for example, `process.name : "mctray.exe"` for a McAfee security binary). Then, search the results for the `process.code_signature.subject_name` field, which contains the signer's name (for example, `McAfee, Inc.`).
+
+.. `Operator`: The operator is `is one of`, and it cannot be modified.
+
+.. `Value`: Enter the hash value, file path, or signer name. To enter multiple values (such as a list of known malicious hash values), you can enter each value individually or paste a comma-delimited list, then press **Return**.
++
+NOTE: Hash values must be valid to add them to the blocklist. In addition, to minimize visibility gaps in the {security-app}, be as specific as possible in your entries. For example, combine `Signature` information with a known `Path`.
```

Comment:
Excellent catch -- I copied a lot of these steps from the trusted applications feature because they're so similar, and I should have cut this out. Trusted apps basically do the opposite of blocklist, hence why this seems (rightly) backwards to you!
```suggestion
NOTE: Hash values must be valid to add them to the blocklist.
```